frontend/src/templates/index.mustache

```diff
@@ -11,7 +11,7 @@
     <meta name="viewport" content="width=device-width">
     <link rel="localization" href="/static/locales/{locale}/app.ftl">
 
-    <link rel="canonical" href="https://testpilot.firefox.com/">
+    <link rel="canonical" href="https://testpilot.firefox.com/{{{ canonical_path }}}">
```

Comment:
Yeah, I think that's an artifact of the [connect-history-api-fallback](https://github.com/mozilla/testpilot/blob/master/frontend/tasks/server.js#L13) module showing the root `index.html`. If you add a trailing slash, you should see something different - i.e. `min-vid/'